README.md

```diff
@@ -137,3 +137,5 @@ meeting invite and Zoom links to join the meeting.
 [kong-logo]: https://konghq.com/wp-content/uploads/2018/05/kong-logo-github-readme.png
 [k4k8s-enterprise-setup]: https://docs.konghq.com/kubernetes-ingress-controller/latest/deployment/k4k8s-enterprise/
 [guides]: https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/overview/
+
+### migrate knative ingress supoort from v1.3.x to v2.0. 1
```

Comment:
It would appear we intended to add some documentation here which covers the effects on knative.Ingress when upgrading to v2.0, we need to finish this documentation (or remove it if it's no longer relevant).

Reply:
I have been understanding this concise documentation should be moved to Changelog.md ?

Reply:
Sure, but we also seem to need to expand upon it because what I'm seeing currently just looks like a header, no?

Reply:
A clear documentation (what is added) should be able to satisfy the need.